LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) Caroline
+Copyright (c) Maxime U Garcia
```

Comment:
yeah, we can update that

Reply:
Done in https://github.com/nf-core/riboseq/pull/50